src/goto-symex/symex_builtin_functions.cpp

```diff
@@ -177,8 +177,8 @@ void goto_symext::symex_allocate(
   }
   else
   {
-    exprt nondet = build_symex_nondet(object_type);
-    code_assignt assignment(value_symbol.symbol_expr(), nondet);
+    const exprt nondet = build_symex_nondet(object_type, nondet_count);
+    const code_assignt assignment(value_symbol.symbol_expr(), nondet);
```

Comment:
I don't think this change works in here, given the comments that have been removed from the PR.

Reply:
yes thanks, it wasn't caught while rebasing.